delete deprecated clip example

ramyaparimi opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

<!-- ✍️ Summarize your changes in one or two sentences -->

`clip` is deprecated; deleted it from the CSS example in this page https://developer.mozilla.org/en-US/docs/Web/API/File_API/Using_files_from_web_applications 

fixes https://github.com/mdn/content/issues/35743

### Motivation

<!-- ❓ Why are you making these changes and how do they help readers? -->

I have learnt so much from MDN docs and want to contribute back. Deleting deprecated examples with help reader learn relevant stuff.

ramyaparimi committed:
delete deprecated clip example

Josh-Cena reviewed (dismissed):
This makes this input actually visible because it has 1px size. Have you checked out the suggestion in https://github.com/mdn/content/issues/35743#issuecomment-2330671273?

ramyaparimi committed:
Update index.md

adding modern `clip-path: inset(50%);` property

ramyaparimi committed:
Update files/en-us/web/api/file_api/using_files_from_web_applications/index.md

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

ramyaparimi commented:
I guess I was over ambitious by taking up this task 😞 . I hope I understood the context better this time. I am still not very confident with CSS styling.

Josh-Cena commented:
Perhaps @estelle could say more about best practices here.

ramyaparimi commented:
So I tested both stylings, one in the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/File_API/Using_files_from_web_applications#using_a_label_element_to_trigger_a_hidden_file_input_element) and other from the [A11Y project](https://www.a11yproject.com/posts/how-to-hide-content/). They both seem to have the same output but both of them use `clip` feature.

So next, I went to the [clip doc on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/clip) to see any suggestions are made on how to replace clip. This suggested to use the [new `clip-path` property](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path), which is mentioned in the A11Y project and the [comment in the issue](https://github.com/mdn/content/issues/35743#issuecomment-2330671273). 

So I then tested the code removing `clip` but keeping the `clip-path: inset(50%);` property, and the outcome seems to be same as with `clip` property. 

So I went and changed the CSS styling by completely removing `clip` property and keeping the `clip-path: inset(50%)` property. 

However, after doing all these tests got me curious as to why to use any `clip` property at all. Because when I removed `clip` and `clip-path` from styling, the outcome was still the same. There was no difference at all 🤔 . 

Here are the screenshots.

**With `clip` as in [MDN docs:](https://developer.mozilla.org/en-US/docs/Web/API/File_API/Using_files_from_web_applications#using_a_label_element_to_trigger_a_hidden_file_input_element)**

<img width="969" alt="image" src="https://github.com/user-attachments/assets/b811b3a4-c42a-4bb6-a2ab-d83859ea7378">

**with new `[clip-path](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path)`**
<img width="973" alt="image" src="https://github.com/user-attachments/assets/6fcd4389-97df-4461-b268-e0663ec269a4">


**Without both `clip` and `clip-path`**

<img width="989" alt="image" src="https://github.com/user-attachments/assets/adcea226-2618-4b5b-b77b-d940994ca77a">


Please let me know if I am missing anything in my understanding. I am still not great with CSS and all its complexities 🙈 .

ramyaparimi committed:
Update index.md

add clip-path remove clip

estelle committed:
added clip

estelle reviewed (approved):
Thank you. Also, congratulations on your first merged MDN content PR. Welcome to the team!
